packages/terra-core-docs/src/terra-dev-site/doc/show-hide/About.1.doc.mdx

```diff
@@ -42,9 +46,113 @@ The Show-Hide component must be composed inside the [Base][1] component with a l
+
+<Notice variant="important" ariaLevel="3">
+
+  To make the Show/Hide component accessible, you must use the `focusRef` prop. This prop allows assistive technologies such as screen readers to put focus on the hidden text and announce it when it is disclosed. The `focusRef` is optional for backward compatibility but is required to comply with accessibility requirements.
+
+</Notice>
+
+#### Truncate a paragraph of text
+  - At the top level of your component declare a `ref`, pass it as `focusRef` props to `ShowHide` component. 
+  - Pass a preview node with the portion of the text as a prop to `ShowHide` component.
+  - Use `Focuser` component inside `<p>` tag. A `Focuser` component allows `ShowHide` component to set focus on the beginning of the focusable text upon opening. 
+  - The text splitted in the prefix and focusable text portions has to be passed as string props. The prefix prop should contain the portion of the text that was a part of the preview prop, passed to `ShowHide`.
+  - Forward `focusRef` to `Focuser` component.
+
+  ```jsx
+  import React, { useRef, useState } from 'react';
+  import ShowHide, { Focuser } from 'terra-show-hide';
+
+  const MyComponent = () => {
+    const focusRef = useRef(null);
+    const [isOpen, setIsOpen] = useState(false);
+    const toggleShowHide = () => setIsOpen(!isOpen);
+
+    const prefix = 'Patients are requesting greater affordability and efficiency in healthcare. With procedures performed in an ambulatory surgery center costing up to 60% less compared to a hospital outpatient department (1), the demand for these facilities is increasing.';
+    const focusableText = 'In fact, the U.S. ambulatory surgery center market is expected to see a 6.9 % compound annual growth rate, reaching $33 billion by 2028. (2) Cerner understands the urgency to grow in the ambulatory surgery center market while continuing to deliver excellent care. Healthcare IT products can help improve clinician efficiency and patient outcomes, as well as enhance communication and data exchange between ambulatory surgery center providers and patients. Read';
+
+    return (
+      <ShowHide focusRef={focusRef} preview={ <p>{prefix}</p> } isOpen={isOpen} onChange={toggleShowHide} >
+        <p>
+          <Focuser ref={this.focusRef} prefix={prefix} focusableText={focusableText} />
+        </p>
+      </ShowHide>
+    );
+  }
+  ```
+
+#### Multiple children
+  Even though the main purpose of the `ShowHide` component is to truncate long paragraps of texts, the content of the `ShowHide` does not have to be one paragraph. It can include links, lists, and any king of HTML tags as long as the first line of the hidden text is passed as a string prop to `Focuser` component rather than an HTML element. Please note:
+  - Links can be placed inside the `p` tag that wrapps `Focuser` component both preceding and following it. The prop `prefix` can be omitted if content has been passes as an HTML element preceding `Focuser`.
```

Comment:
```suggestion
  Even though the main purpose of the `ShowHide` component is to truncate long paragraps of texts, the content of the `ShowHide` does not have to be one paragraph. It can include links, lists, and any kind of HTML tags as long as the first line of the hidden text is passed as a string prop to `Focuser` component rather than an HTML element.
  
Please note:
  - Links can be placed inside the `p` tag that wraps `Focuser` component both preceding and following it. The prop `prefix` can be omitted if content has been passed as an HTML element preceding `Focuser`.
```